sns/src/main/scala/akka/stream/alpakka/sns/SnsPublishFlowStage.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.sns
+
+import akka.stream._
+import akka.stream.stage._
+import com.amazonaws.handlers.AsyncHandler
+import com.amazonaws.services.sns.AmazonSNSAsync
+import com.amazonaws.services.sns.model.{PublishRequest, PublishResult}
+
+private[akka] final class SnsPublishFlowStage(topicArn: String, snsClient: AmazonSNSAsync)
```

Comment:
NVM, I already asked that previously but github hid all comments and I had forgot.